doc/source/how-to/documenting.rst

```diff
@@ -68,14 +68,13 @@ Directory` directory. The ``index.rst`` file in the ``doc/source`` directory
 defines the first level of your documentation hierarchy.  The ``toctree``
 directive (which stands for "table of contents tree") indicates the maximum
 number of heading levels that the documentation is to display. Below this
```

Comment:
```suggestion
number of heading levels that the documentation is to display. Following this
```

Reply:
The Vale linter, when implemented, will flag above and below as errors--so I'm suggesting we fix it now.